docs/developer-docs/daos/nns/concepts/proposals/verify-proposals.mdx

```diff
@@ -0,0 +1,198 @@
+ For proposals that add or remove node providers, you can check that
+ * there is a proper forum introduction by the new NP
+ * self-declaration and Identity documents are uploaded on the wiki
+ * hashes of the self-declaration and identity document match the hashes in the proposal
+ * if this is a new region or community, that there is an entry in the public chamber of commerce register  (if chamber
+  of commerce number is provided)
+
+### Add or remove Data Centers
+For proposals that add or remove data centers, you can
+  * check who submitted the proposal (normally only a new node provider)
+  * perform an Internet search whether this is a legitimate data center.
+
+## Verify Node Admin proposals
+If you are looking to verify [node admin proposals](/docs/current/developer-docs/daos/nns/concepts/proposals/proposal-topics#Topic:-`NodeAdmin`),
+these are some steps you can consider for verification.
+
+### Update node configuration
+In such a proposal, a node provider sets the number of nodes and the `node_type` (Gen1, Gen2).
+You can
+* check these against the originally approved node allowance (whether that allowance was approved, and whether it
+matches the IC Target topology using the [github tooling](https://github.com/StephanusFischer/decentralization)
+
+### Add node operator
+ This proposal sets the allowance for a particular node provider in a specific data center.
+ You can perform the following checks
+ * if it matches the IC Target topology using the [github tooling](https://github.com/StephanusFischer/decentralization). For example, such proposals should be rejected if the target topology has
+ been reached.
+ * whether the node provider-id and data center-id match with the previously submitted proposals (the add node provider proposal, and the add data center proposal).
+
+### Remove nodes from the network
+ This is a proposal to remove (unhealthy) nodes from the network.
+ You can
+ * check whether the nodes are indeed unassigned and not in a particular subnet.
+
+
+## Verify Subnet management proposals
+If you are looking to verify [subnet management proposals](/docs/current/developer-docs/daos/nns/concepts/proposals/proposal-topics#Topic:-`SubnetManagement`), these are some steps you can consider for verification.
+
+### Node replacement proposals
+ For these proposals you can
+ * check whether the nodes are replaced because they are dead/unhealthy or replaced for other good reasons.
+
+### Other proposals of this topic
+You can
+ * check whether the reasoning in the proposal makes sense wrt which proposals need to be adopted in which order.
+
+
+## Verify service nervous system management proposals
+If you are looking to verify [service nervous system management proposals](/docs/current/developer-docs/daos/nns/concepts/proposals/proposal-topics#Topic:-`ServiceNervousSystemManagement`),
+you can refer to this video that explains the process on the example of an NNS proposal that elects a new SNS
+governance wasm:
```

Comment:
```suggestion
## Verify subnet management proposals
If you are looking to verify [subnet management proposals](/docs/current/developer-docs/daos/nns/concepts/proposals/proposal-topics#Topic:-`SubnetManagement`), these are some steps you can consider for verification.

### Node replacement proposals
You can:
* Check whether the nodes are replaced because they are dead/unhealthy, or replaced for other good reasons.

### Other proposals of this topic
You can:
* Check whether the reasoning in the proposal makes sense with regard to which proposals need to be adopted in which order.

## Verify service nervous system management proposals
If you are looking to verify [service nervous system management proposals](/docs/current/developer-docs/daos/nns/concepts/proposals/proposal-topics#Topic:-`ServiceNervousSystemManagement`),
you can refer to this video that explains the process using an example of an NNS proposal that elects a new SNS
governance Wasm:
```